setup.py

```diff
@@ -3,7 +3,8 @@
 
 setup(
     name='dynamodb_utils',
-    version='1.0.0',
+    version='1.1.0',
```

Comment:
please don't bump version number, that's done at release time when reviewing the changelog. For example I don't think this deserves a minor version change even because the public facing API hasn't changed in any way, users will still just run `dynamodb-dumper`